packages/terra-application/package.json

```diff
@@ -45,11 +45,14 @@
   },
   "dependencies": {
     "@cerner/terra-docs": "^1.0.0",
+    "@formatjs/intl-relativetimeformat": "^8.0.0",
     "classnames": "^2.2.5",
     "eventemitter3": "^4.0.4",
     "focus-trap-react": "^6.0.0",
     "keycode-js": "^3.1.0",
     "lodash.debounce": "^4.0.8",
+    "intl": "^1.2.5",
+    "intl-locales-supported": "^1.8.4",
```

Comment:
We should update this to `">=1.8.4 <1.8.12 || ^1.8.13"` to avoid the 1.8.12 because it was a bad publish. See https://github.com/cerner/terra-core/pull/3344